conda_pip/data/EXTERNALLY-MANAGED

```diff
@@ -0,0 +1,11 @@
+[externally-managed]
+Error=You are trying to modify a conda environment!
+  
+  If you want to install PyPI packages into this conda environment,
+  please consider using 'conda pip install ...' instead. 
+  This will make a few checks to ensure that the PyPI packages 
+  you want to install will not disrupt your conda environment.
+
+  If you still want to proceed with 'pip install ...', then
+  add the '--break-system-packages' to your command, but be aware
+  that you might experience unexpected issues.
```

Comment:
`PIP_BREAK_SYSTEM_PACKAGES` or the config option might also be worth mentioning. Or linking to the pip docs about it?

Reply:
Adding both in the next push.